### PR TITLE
Script for defect 2467

### PR DESCRIPTION
**Pull request by TMyza**
ATF Test Scripts to check [#2467](https://github.com/smartdevicelink/sdl_core/issues/2467)

Summary
A set of test scripts to verify that issue is fixed

This PR is **not ready** for review.
This is **draft** version of the script.

### CODE REVIEW

File: test_scripts/Defects/5_0/2467_Policy_table_corupted_after_first_ign_cycle.lua [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    test.mobileSession[1] = nil
+    utils.wait()
+end
+
+local function pTUpdateFunc(tbl)
+    tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.fullAppID].groups = { "Base-4", "SendLocation" }
+end
+
+local function sendLocation()
+    local cid = common.getMobileSession():SendRPC("SendLocation", {
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/2467_Policy_table_corupted_after_first_ign_cycle.lua]: Remove redundant white space in the end of line.

File: test_scripts/Defects/5_0/2467_Policy_table_corupted_after_first_ign_cycle.lua [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS",{})
+    end)
+    common.getMobileSession():ExpectResponse(cid, { success = true , resultCode = "SUCCESS" })
+end
+
+local function ignitionOff()
+    common.getHMIConnection():SendNotification("BasicCommunication.OnExitAllApplications", { reason = "SUSPEND" })
+    common.getHMIConnection():ExpectNotification("BasicCommunication.OnSDLPersistenceComplete")
+    :Do(function()
+        sdl:DeleteFile()
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/2467_Policy_table_corupted_after_first_ign_cycle.lua]: Please remove ` sdl:DeleteFile()`

File: test_scripts/Defects/5_0/2467_Policy_table_corupted_after_first_ign_cycle.lua [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+-- 4) Register app.
+-- Expected result:
+-- Permissions should not be lost.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('user_modules/sequences/actions')
+local runner = require('user_modules/script_runner')
+local utils = require("user_modules/utils")
+local test = require("user_modules/dummy_connecttest")
+local sdl = require("SDL")
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/2467_Policy_table_corupted_after_first_ign_cycle.lua]: remove local sdl = require("SDL") with `sdl:DeleteFile()`

File: test_scripts/Defects/5_0/2467_Policy_table_corupted_after_first_ign_cycle.lua [hunk 4]
```diff
@@ -0,0 +1,82 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2467
+--
+-- Description:
+-- Policy table corupted after first ign cycle.
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/2467_Policy_table_corupted_after_first_ign_cycle.lua]: corupted -> corrupted

File: test_scripts/Defects/5_0/2467_Policy_table_corupted_after_first_ign_cycle.lua [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+runner.Step("Send Location", sendLocation)
+runner.Step("IGNITION_OFF", ignitionOff)
+runner.Step("Clean sessions", cleanSessions)
+
+-- [[ Test ]]
+runner.Step("IGNITION_ON", common.start)
+runner.Step("Register App", common.registerApp)
+runner.Step("Send Location after IGNITION_ON", sendLocation)
+
+
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/2467_Policy_table_corupted_after_first_ign_cycle.lua]: remove redundant new line

**Comment by dboltovskyi** [comment]
Livio team please do not review this PR for now since it hasn't passed our internal review yet.  
Review and appropriate updates going be done till 1st of November.
Thank you.

**Comment by TMyza** [comment]
@GetmanetsIrina , added changes according to comments to PR, please review. 
Commit (b50e963)

**Comment by dboltovskyi** [comment]
The internal review iteration has been moved to the LuxoftSDL.

**Pull request closed without merging**